src/components/MessagesList/MessagesGroup/MessagesGroup.vue

```diff
@@ -32,7 +32,7 @@
 		<ul class="messages">
 			<li class="messages__author" aria-level="4">
 				{{ actorDisplayName }}
-				<div v-if="lastEditActorDisplayName">
+				<div v-if="lastEditTimestamp">
```

Comment:
`lastEditActorDisplayName` is more important to check here

Reply:
I don't want to rely on the displayName, because it could be changed to empty stting -> evaluated to `false`. In that sense, timestamp is more reliable

Reply:
Although the lastEdit parameters haven't changed for deleted user, but that's on a backend side:
![image](https://github.com/nextcloud/spreed/assets/93392545/d26ace7e-f196-4fb0-8522-fb05b42247d0)